product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,213 @@
+ ```
+
+The script displays the following:  
+
+* A list of required Azure Service Providers and their registeration status. Ensure that you register the service providers that are displayed as *NotRegistered* in the *Registeration state* column. See [Register Azure service providers](#register-azure-service-providers)
+
+```
+ __OUTPUT__
+
+#######################
+# Provider            #
+#######################
+
+Namespace                                RegistrationPolicy    RegistrationState    ProviderAuthorizationConsentState
+---------------------------------------  --------------------  -------------------  -----------------------------------
+Microsoft.Capacity                       RegistrationRequired  NotRegistered
+Microsoft.ContainerInstance              RegistrationRequired  NotRegistered
+Microsoft.Compute                        RegistrationRequired  NotRegistered
+Microsoft.ContainerService               RegistrationRequired  NotRegistered
+Microsoft.KeyVault                       RegistrationRequired  NotRegistered
+Microsoft.ManagedIdentity                RegistrationRequired  NotRegistered
+Microsoft.Network                        RegistrationRequired  NotRegistered
+Microsoft.OperationalInsights            RegistrationRequired  NotRegistered
+Microsoft.OperationsManagement           RegistrationRequired  NotRegistered
+Microsoft.Portal                         RegistrationFree      Registered
+Microsoft.Storage                        RegistrationRequired  Registered
+Microsoft.AlertsManagement               RegistrationRequired  NotRegistered
+
+```
+ 
+ 
+* Whether you have sufficient limits on vCPUs for the `Standard_D2_v4` and `Standard_E2s_v3` VM size families in your region (and availability zone, if HA is enabled). Open a support request to remove SKU restrictions for the VM families with *NotAvailableForSubscription* displayed in the *Restrictions* column. See [Fix issues with SKU restrictions](#fix-issues-with-sku-restrictions)
+
+```
+ __OUTPUT__
+  
+  #######################
+# Virtual-Machine SKU #
+#######################
+
+ResourceType      Locations              Name                    Zones    Restrictions
+------------      ---------              ----                    -----    ------------
+virtualMachines   eastus2                Standard_D2_v4          1,2,3    None
+virtualMachines   eastus2                Standard_E2s_v3         1,2,3    NotAvailableForSubscription, type: Zone, locations: eastus2, zones: 1,3
+
+```
+
+* Whether you have sufficient limits on vCPUs and IP addresses for your region. Open a support request to raise limits for the vCPUs and IP addresses if they exceed the available VM families with *NotAvailableForSubscription* displayed in the *Restrictions* column. See [Increase Public IP addresses](#increase-public-ip-addresses-limits) and [Increase vCPU limits](#increase-vcpu-limits)
```

Comment:
* Whether you have sufficient limits on vCPUs and IP addresses for the region in your Azure subscription. Open a support request to raise limits for the vCPUs and IP addresses if the *Suggestion* column of *Quota Limitation* table asks for *Need Increase*. See [Increase Public IP addresses](#increase-public-ip-addresses-limits) and [Increase vCPU limits](#increase-vcpu-limits)

Reply:
Done.